docs/development.md

```diff
@@ -57,6 +57,55 @@ yarn schema:drop
 yarn migration:generate migration_name
 ```
 
+#### MySQL
+
+If you need to use MySQL / MariaDB instead of PostgreSQL, follow the steps below:
+> (assuming you have installed mysql in your system and it is running on port 3306)
+1. Make the following entries in the #DB section in `.env` file
+
+```env
+DB_TYPE=mysql
+DB_HOST=127.0.0.1
+DB_PORT=3306
+DB_USERNAME=mysql
+DB_PASSWORD=mysql
+DB_DATABASE=nest_boilerplate
+MYSQL_DATABASE=nest_boilerplate
+MYSQL_USER=mysql
```

Comment:
@SaadBazaz let's remove MYSQL_* envs and use only DB_*

Reply:
Good call.